src/sentry/static/sentry/app/components/checkboxFancy.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+import InlineSvg from 'app/components/inlineSvg';
+
+class CheckboxFancy extends React.Component {
+  static defaultProps = {
+    checked: false,
+  };
+
+  render() {
+    return (
+      <CheckboxContainer
+        role="checkbox"
+        aria-checked={this.props.checked}
+        checked={this.props.checked}
+        {...this.props}
+      >
+        {this.props.checked && <Check src="icon-checkmark-sm" />}
+      </CheckboxContainer>
+    );
+  }
+}
+
+const CheckboxContainer = styled('div')`
+  width: 18px;
```

Comment:
should this be a size property?

Reply:
I tried it at 16 (`space(2)`) and it felt too small. I tried it at 20 (`space(3)`) and it felt a little too big. 

Ultimately, I don't think it's as important for width and height stuff. We should use it when we can, but it's more valuable to get it right.

Reply:
Sorry, I meant as a component property (not space), so that we can potentially have differently sized fancy checkboxes